src/test/java/teammates/test/cases/storage/FeedbackResponseCommentsDbTest.java

```diff
@@ -41,9 +41,9 @@ public void classSetup() throws Exception {
         frcDb.createEntity(frcaData);
         frcDb.createEntity(anotherFrcaData);
         frcaData = frcDb.getFeedbackResponseComment(frcaData.feedbackResponseId,
-                                 frcaData.giverEmail, frcaData.createdAt);
+                                 frcaData.commentGiver, frcaData.createdAt);
```

Comment:
Test for the newly added method?

Reply:
Which one? I didn't add a new method I guess.

Reply:
Not in this file maybe. I believe you have changed some DB files, you need either edit the existing tests or add new tests.